pkg/kubelet/nodestatus/setters.go

```diff
@@ -109,7 +109,11 @@ func NodeAddress(nodeIPs []net.IP, // typically Kubelet.nodeIPs
 			if node.ObjectMeta.Annotations == nil {
 				node.ObjectMeta.Annotations = make(map[string]string)
 			}
-			node.ObjectMeta.Annotations[cloudproviderapi.AnnotationAlphaProvidedIPAddr] = nodeIP.String()
+			annotation := nodeIP.String()
+			if len(nodeIPs) == 2 {
```

Comment:
do you want to reuse the `secondaryNodeIPSpecified` and `secondaryNodeIP` ? it may be more readible

Reply:
indeed. done